### DISCUSSION TITLE
Q: Waline 评论区不显示 || Q: Waline’s comment area is not displayed

### DISCUSSION THREAD

**Discussion by xingteh** [walinejs/waline#2265] [Q&A] [answered]
### Describe the bug | 问题描述

最近博客在进行大更新，想用 Waline 但是不显示评论框。
排查了很多，Twikoo giscus 和 Gitalk 都能显示。

### Website URL | 问题网站

https://blog.xinxin2021.tk/zatan_1/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

The blog is undergoing a major update recently, and I want to use Waline but the comment box does not appear.
After a lot of troubleshooting, Twikoo giscus and Gitalk can display it.

### Website URL | Problem website

https://blog.xinxin2021.tk/zatan_1/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
可以提供一个可复现的页面，我帮忙看下

---
Can you provide a reproducible page and let me take a look

**Reply by xingteh** [reply]
https://blog.xinxin2021.tk/comments/

麻烦帮忙看下，万分感谢

---
https://blog.xinxin2021.tk/comments/

Please help me take a look, thank you very much

**Reply by lizheming** [reply]
进去是 404 页面

---
Entering is a 404 page

**Reply by xingteh** [reply]
sry 多敲一个斜杠

https://blog.xinxin2021.tk/comments

**Reply by lizheming** [reply, marked as answer]
这一行少了个逗号

![image](https://github.com/walinejs/waline/assets/424491/c320fb98-2c18-4d88-b469-c36bcd2fc77d)
